tests/json_test/json_test.cpp

```diff
@@ -5297,26 +5302,39 @@ struct glz::meta<custom_encoding>
    using T = custom_encoding;
    static constexpr auto value = object("x", custom<&T::read_x, &T::write_x>, //
                                         "y", custom<&T::read_y, &T::y>, //
-                                        "z", custom<&T::z, &T::write_z>);
+                                        "z", custom<&T::z, &T::write_z>, //
+                                        "w",
+                                        custom<
+#if defined(__GNUC__) && (!defined __clang__)
+                                           &T::read_w
+#else
+                                           &T::w
+#endif
```

Comment:
What's the specific feature clang doesn't offer that prevents `read_w` as a non-type template parameter?